python/gps/algorithm/traj_opt/config.py

```diff
@@ -6,13 +6,28 @@
     # Dual variable updates for non-PD Q-function.
     'del0': 1e-4,
     'eta_error_threshold': 1e16,
-    'min_eta': 1e-4,
+    'min_eta': 1e-8,
```

Comment:
It doesn't as far as I can tell, none of the other experiments I've tried seemed to come close to the old `min_eta` value.